.github/workflows/deploy.yml

```diff
@@ -34,7 +34,7 @@ jobs:
                   npm run beachball-bump
 
             - name: Build
-              run: npm run tsc
+              run: npm run build
```

Comment:
Do we still need this deploy workflow in addition to the release workflow?

Reply:
we will not, but i didn't want to remove it just yet in case we needed to do a release before finishing validation of the new process. so be-svcs will still trigger deploy for now until the new release workflow is merged into all the sdks.

Reply:
This workflow won't run without beachball in this repo

Reply:
🤦 added it back in